run-platform.sh

```diff
@@ -322,6 +337,7 @@ do_git_pull
 setup_env
 build_services
 run_services
+show_migration_warning
```

Comment:
@chandrasekharan-zipstack Better to rename to `show_release_warnings`, to make it generic as mentioned above.